Esri/arcgis-maps-sdk-swift-samples
# [New] Show scale bar

## Description

This PR implements **`Show scale bar `** in `Maps` category.

`swift/issues/5465`

## How To Test

Load map and check that the scale bar is present on the bottom of the screen. Zoom in on location on map and check to see that the scale is adjusted for the level of zoom. 

## Screenshots
![show-scale-bar](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/400a62ca-a4f1-45a7-b6e8-1e9b55e326c5)
